packages/core/src/internal/utils/global.ts

```diff
@@ -11,35 +11,26 @@ import { LogService } from '../services/log.service.js';
 
 export interface CDSGlobal {
   _version: string[];
-  _loadedElements: string[];
   _react: { version: string }; // set by @cds/react
   _supports: FeatureSupportMatrix;
-  getVersion: () => CDSLog;
+  _stateProxy: boolean;
+  _state: Readonly<CDSState>;
+  getVersion: () => any;
   logVersion: () => void;
   environment: {
     /** Set to true for production env to disable dev time logging and tooling */
     production: boolean;
   };
-  state?: CDSState;
-}
-
-export interface CDSLog {
-  versions: string[];
-  loadedElements: string[];
-  userAgent: string;
-  supports: {};
-  angularVersion?: string | undefined;
-  angularJSVersion?: string | undefined;
-  reactVersion?: string | undefined;
-  vueVersion?: string | undefined;
-  environment: {
-    production: boolean;
-  };
 }
 
 export interface CDSState {
-  focusTraps?: string[];
+  focusTrapRegistry: Readonly<{ focusTrapId: string }[]>;
```

Comment:
Does it make sense to have all the items here be `*Registry`? Maybe this interface isn't `CDSState` but `CDSRegistries`? The keys in the interface seem to be compensating for the name or architecture of `CDSState` not being descriptive here.

Reply:
Yeah you are right.  I don't think they all need to be the same. The focusTrap isnt really a registry. Its not registering anything like icons/i18n where they are registered for later lookups. Its really just in memory state.